app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
+
```

Comment:
Please remove your express.json. It's not necessary until you reach a post request so it shouldn't be in here at this point 🙂